modules/rosapps/applications/net/tsclient/rdesktop/uiports/qtewin.cpp

```diff
@@ -922,7 +922,7 @@ void QMyMainWindow::MemuClicked(int MenuID)
     rdp_send_input(0, RDP_INPUT_SCANCODE, RDP_KEYRELEASE, 0x1d, 0); // control
     rdp_send_input(0, RDP_INPUT_SCANCODE, RDP_KEYRELEASE, 0x38, 0); // alt
   }
-  else if (MenuID == 4) // double click
+  else if (MenuID == 4) // double-click
```

Comment:
This is 3rd-party code, please send the fix upstream.

https://git.reactos.org/?p=reactos.git;a=blob;f=media/doc/3rd+Party+Files.txt;hb=50774f865c4f9d9b926f32a8bcfc20bbeced8fb5#l9